dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/PropertyUtils.java

```diff
@@ -88,6 +88,9 @@ public static boolean getResUploadStartupState() {
      * @return property value
      */
     public static String getString(String key) {
+        if (key == null) {
```

Comment:
May i ask in which scenario will there be a null key?

Reply:
When the property key is read from database, or some other place..